cli.js

```diff
@@ -59,6 +59,10 @@ const cli = meow(help, {
     move: {
       type: 'boolean',
       alias: 'm'
+    },
+    clean: {
+      type: 'boolean',
+      alias: 'C'
```

Comment:
Considering that this a console app and the fact that `clear` is a widely-used and familiar command among CLI users, with a closely similar effect to that of the one here, it (clear) seems to be a more appropriate flag name than `clean`.  Also, let's completely avoid setting a shortcut/alias, in order to prevent any accidental uses.

```
clear: {
  type: 'boolean'
}
```